student-store-ui/src/components/App/AppOverlay.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import Sidebar from '../Sidebar/Sidebar'
+import './AppOverlay.css'
+import { Outlet } from 'react-router-dom'
+
+function AppOverlay({emptyQuantities, clearCart, cart, setCart, quantities, setQuantities, originalProducts}) {
+    return (
+        <div className='app-overlay'>
+           
+            <Sidebar emptyQuantities={emptyQuantities} clearCart={clearCart} cart={cart} setCart={setCart} className='sidebar' quantities={quantities} setQuantities={setQuantities} originalProducts={originalProducts}/>
```

Comment:
format this using multiple lines (best practice is limit 80-characters per line)